frontend/src/pages/projects/projectSharing/data.ts

```diff
@@ -0,0 +1,81 @@
+import { RoleBindingKind } from '~/k8sTypes';
+import { SortableData } from '~/utilities/useTableColumnSort';
+import { compareDatesWithUndefined, ensureRoleBindingCreationSorting } from './utils';
+
+export const columnsProjectSharingUser: SortableData<RoleBindingKind>[] = [
+  {
+    field: 'username',
+    label: 'Username',
+    width: 30,
+    sortable: (a, b, key, sortDirection) =>
```

Comment:
Mmmm maybe I should left some comments right there, this is targeted for when you are adding a new row, that row is created at the bottom of the table and it's suuuper weird that if you sort the table it will be sorted out at the top of the table, in order to sort all the rows except that, I used this.

Maybe is not super clear or maybe is not needed, but we can discuss it later.

Reply:
Hmmm -- maybe we add a footer row concept to the base table 🤔 cc @Gkrumbach07 

I think if we had a means to just always "place a row at the bottom of the table" -- it'll allow for other creates like this. Maybe we can find other uses over the course of time.

Thoughts?

Reply:
Ok, if we can add a sticky footer cell can be a better solution rather than this semi-workaround. At first I was aiming for something like that, but I didn't find anything similar in patternfly, I'm not sure about the UX input here either, but for me it felt weird that the creation row can be rearranged before added as a new element.

Reply:
@Gkrumbach07 For more context, I'll link here the gif referring to the visual change ![sort](https://user-images.githubusercontent.com/16117276/234551267-bdbf5cd4-a3bf-486c-b160-fd07d22a2d18.gif)

Reply:
A footer row makes sense, but wouldn't the row get moved to a sorted position after form completion aka hitting the check mark?